TDL-06-EC2-Cloudwatch-S3/terraform-code-ec2-cw-s3/scripts/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Upload the config file to the bucket
+aws s3 cp ./config.json s3://tdl6-bucket/config.json
+
```

Comment:
Try to avoid hardcoded value and make ur code reusable:)
Option 1 - define  bucket name at the very beginning of the script and refer the variable later:
bucketName='tdl6-bucket'
aws s3 cp ./config.json s3://${bucket)Name}/config.json


Option 2: (advanced level, read bucket name from variable.tf
bucketName=$(sed -ne '/bucket_name/,+5 p' variables.tf |grep default| awk '{print $3}' | tr -d '"')
aws s3 cp ./config.json s3://${bucket)Name}/config.json

And to make the script more flexible:
1. put all source configuration files under one specific folder such as upload
2. Run for loop to upload all files to S3 bucket using aws s3 sync
if [ -d "upload"]; then
    aws s3 sync upload s3://${bucketName}
fi